code/modules/food/recipes_microwave_ch.dm

```diff
@@ -15,60 +14,81 @@
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/bigos
 
-/datum/recipe/concha
-	reagents = list("flour" = 5, "sugar" = 5, "berryjuice" = 5)
+/datum/recipe/microwave/concha
+	reagents = list("berryjuice" = 5)
 	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/dough,
 		/obj/item/weapon/reagent_containers/food/snacks/egg
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/concha
 
-/datum/recipe/pandenata
+/datum/recipe/microwave/pandenata
 	reagents = list("sugar" = 10, "cream" = 5)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/doughslice,
 		/obj/item/weapon/reagent_containers/food/snacks/doughslice,
-		/obj/item/weapon/reagent_containers/food/snacks/doughslice,
-		/obj/item/weapon/reagent_containers/food/snacks/egg
+		/obj/item/weapon/reagent_containers/food/snacks/doughslice
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/pandenata
 
-/datum/recipe/tocino
-	reagents = list("sodiumchloride" = 5, "wine" = 5, "sugar" = 5)
+/datum/recipe/microwave/tocino
+	reagents = list("sodiumchloride" = 5, "wine" = 5)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/rawcutlet
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/tocino
 
-/datum/recipe/garlicbread
-	reagents = list("sodiumchloride" = 2, "blackpepper" = 2)
+/datum/recipe/microwave/garlicbread
 	fruit = list("onion" = 1)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/cheesewedge,
-		/obj/item/weapon/reagent_containers/food/snacks/slice/bread,
 		/obj/item/weapon/reagent_containers/food/snacks/slice/bread
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/garlicbread
-	
+
+/datum/recipe/microwave/plumpburger
+	fruit = list("plumphelmet" = 1)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/cheesewedge,
+		/obj/item/weapon/reagent_containers/food/snacks/bun
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/plumpburger
+
+/datum/recipe/microwave/wataur
+	reagents = list("water" = 100)
+	result = /obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle/wataur
+
+/datum/recipe/microwave/bigbeanburrito
+	fruit = list("soybeans" = 3)
+	items = list(
+	/obj/item/weapon/reagent_containers/food/snacks/meatburrito,
+	/obj/item/weapon/reagent_containers/food/snacks/rawcutlet,
+	/obj/item/weapon/reagent_containers/food/snacks/cheesewedge
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/slice/bigbeanburrito
+
+/datum/recipe/microwave/supremoburrito
+	fruit = list("soybeans" = 3, "chili" = 1)
+	items = list(
+	/obj/item/weapon/reagent_containers/food/snacks/slice/bigbeanburrito,
+	/obj/item/weapon/reagent_containers/food/snacks/sliceable/cheesewheel,
+	/obj/item/weapon/reagent_containers/food/snacks/tortilla,
+	/obj/item/weapon/reagent_containers/food/snacks/meat
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/sliceable/supremoburrito
+
+/*
+<<<<<<< Updated upstream
+
```

Comment:
Looks like Erik might have forgotten to remove this when doing the upstream synch. Look further on down for more.

Reply:
Yeah, not sure if it's important or not. So I just omitted it out with "*//*" since Nadyr's tea code is in there too.

Reply:
No? That's not even on master. Where did you get this?

Reply:
I gave a possible reason where those lines came from, Raz.

Reply:
Reason is false. Look at master.